db/diff_test.go

```diff
@@ -453,3 +453,229 @@ func TestDiffV1(t *testing.T) {
 		}
 	}
 }
+
+func TestDiffV2(t *testing.T) {
```

Comment:
We could potentially refactor these test methods. Now there are 3 copies of them ¯\_(ツ)_/¯

Reply:
Since we have a very small number of customers that we are in close contact with I'd rather just wait until they have updated and then delete all the version 0 and 1 code and tests. We will need a better sunsetting strategy eventually, but we can do the simple thing now.